src/Model/EnemyManager.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Max
+ * Date: 26/10/20
+ * Time: 9:00
+ */
+
+namespace App\Model;
+
+use PDO;
+
+/**
+ *
+ */
+class EnemyManager extends AbstractManager
+{
+    /**
+     *
+     */
+    const TABLE = 'enemy';
+    const ERROR = -1;
+
+    /**
+     *  Initializes this class.
+     */
+    public function __construct()
+    {
+        parent::__construct(self::TABLE);
+    }
+   
+    public function insertEnemy(Troop $enemy): int
+    {
+        // prepared request
+        $insertEnemy = $this->pdo->prepare("INSERT INTO " . self::TABLE .
+            " (name, strength) VALUES (:name, :strength)");
+        if ((false == $insertEnemy)
+            || (false == $insertEnemy->bindValue('name', $enemy->getName(), PDO::PARAM_STR))
+            || (false == $insertEnemy->bindValue('strength', $enemy->getLevel(), PDO::PARAM_INT))) {
+                return self::ERROR;
+        } else {
+            if ($insertEnemy->execute()) {
+                return (int)$this->pdo->lastInsertId();
+            }
+        }
+        return self::ERROR;
+    }
+    
+    public function deleteAttacker()
+    {
+        $truncate = $this->pdo->prepare("TRUNCATE " . self::TABLE);
+        if (false == $truncate) {
+            return self::ERROR;
+        } else {
+            $truncate->execute();
+        }
+        return "";
```

Comment:
execute() return a boolean.
It may be better for this function to also returns a boolean telling ok or problem
And there's no point returning an empty string
And you should type your functions